startables/readers/bulk.py

```diff
@@ -0,0 +1,76 @@
+import glob
+import os
+from pathlib import Path
+from typing import Union, Iterable
+
+from startables import Bundle, read_csv, read_excel
+
+
+def read_bulk(paths: Union[str, Path, Iterable[Union[str, Path]]]) -> Bundle:
+    """Reads all files from supplied paths and returns a single Bundle containing all table blocks read from all files.
+       Supported extensions are: [csv, xlsx] (soon docx?)
+
+    Arguments:
+        paths {[type]} -- Can be a file, a folder or a glob expression that contains Startables files with supported extensions. Can also be an Iterable of files, folders, and glob expressions.
+
+    Returns:
+        Bundle -- [description]
+
+    NOTES:
+    Multiple tables with the same name (within or across files) are preserved as such in the output
+    bundle.
+    Microsoft Office temp files (starting with "~$" or ending with ".tmp") are ignored when bulk
+    reading a folder or glob expression.
+    """
+
+    if isinstance(paths, str) or isinstance(paths, Path):
+        # Pack single path in something that's iterable, for later convenience
+        paths = [paths]
+
+    def is_temp_garbage(filename: str):
+        """Is this a Microsoft Office temp file?"""
+        return filename.startswith("~$") or filename.endswith(".tmp")
+
+    # First collect the files
+    bulk_files = list()
```

Comment:
I feel like this should be packaged away in a function called `_get_bulk_files()` or similar. (up to line 52)